templates/base.html

```diff
@@ -262,12 +262,24 @@
           >Categories</label>
           <select
             id="categories-list"
-            class="select select-bordered select-sm max-w-xs dm-sans"
+            class="select select-bordered select-sm dropdown max-w-xs dm-sans"
           >
             <option value="">All</option>
             {% for category in categories %}<option value="{{ category }}">{{ category }}</option>{% endfor %}
           </select>
         </div>
+        <div class="text-start ml-[2em] xl:ml-[4em]">
+          <label
+            for="search-bar"
+            class="hidden lg:inline-block kiwix-{{page }}-filter-label text-[0.9em] me-[1em] dm-sans"
+          >Search</label>
+          <input
+            id="search-bar"
+            type="text"
+            placeholder="Search..."
+            class="select select-bordered select-sm max-w-xs dm-sans h-[2.25rem]"
```

Comment:
We cannot have `dm-sans` here as this is free text and users can input any char while dmsans is latin only.